src/test/test_dir.c

```diff
@@ -6573,6 +6573,87 @@ test_dir_dirserv_load_fingerprint_file(void *arg)
   dirserv_free_fingerprint_list();
 }
 
+#define RESET_FP_LIST(list) STMT_BEGIN \
+    dirserv_free_fingerprint_list(); \
+    authdir_init_fingerprint_list(); \
+    list = authdir_return_fingerprint_list(); \
+  STMT_END
+
+#define FP_REJECT 4
```

Comment:
Please include the header that defines this constant.

Reply:
Hmm, there is no header that defines this constant.
Can you call a function that checks if the relay is rejected?

Reply:
I don't think this is possible without adding a lot more code because the function `dirserv_would_reject_router()` expects a `routerstatus_t` and I have a `routerinfo_t` making this impractical.

Reply:
I opened another ticket to make this change:
https://trac.torproject.org/projects/tor/ticket/30677